contracts/VoucherHub.sol

```diff
@@ -300,6 +287,16 @@ contract VoucherHub is
         return voucherAddress.code.length > 0 ? voucherAddress : address(0);
     }
 
+    /**
+     * Get the voucher type details by ID.
+     */
+    function getVoucherType(
```

Comment:
I think voucher type should be moved up a bit since a voucher type should be created before a voucher (so we try to keep a natural order)

Reply:
It's a public view, so how do we want to make the global order ?

Reply:
Ok I see